Server/sprawl_calculator/wup_calculator/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+import uuid
+
+# Create your models here.
+
+class Model(models.Model):
+	id = models.UUIDField() #Id
+	field1 = models.CharField() #Character
```

Comment:
Lets use a actual field name instead of `field 1 `

Reply:
In schemas section in documentation,unable to understand models and their fields.is complete one model.
Here are some fields like id and status with other some 
`from django.db import models
# Create your models here.
class WUPCalculation(models.Model):
        uuid = models.UUIDField(max_length=36, min_length=36, format='uuid', default=uuid.uuid4, unique=True)
        time_value = models.DateTimeField(default=timezone.now)
        latitude = models.DecimalField(max_digits=9, decimal_places=6)
        longitude = models.DecimalField(max_digits=9, decimal_places=6)  
        WUPCalculationRequestStatus = models.CharField(
        max_length=255,
        choices=[
            ('Activating', 'Activating'),
            ('Processing', 'Processing'),
            ('Completed', 'Completed'),
            ('Rejected', 'Rejected'),
            ('Error', 'Error')
        ],
        default='Activating'
    )
`